Lesson32/event-and-state-app/Gokhan/src/components/Exercise10.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+export const Exercise10 = () => {
+
+    useEffect(() => {
+        const dblClickBtn = document.getElementById('dblClickBtn');
+
+        const handleDoubleClick = () => {
+            console.log('The Button Was Double-Clicked');
+        };
+
+        dblClickBtn.addEventListener('dblclick', handleDoubleClick);
+
+        return () => {
+            dblClickBtn.removeEventListener('dblclick', handleDoubleClick);
+        };
+    }, []);
+
+    return (
+        <>
+            <button id="dblClickBtn">Please Double Click The Button</button>
```

Comment:
In this exercise, all you need is to add onDoubleClick event listener to the button
```suggestion
            <button onDoubleClick={handleDoubleClick}>Please Double Click The Button</button>
```

then define handleDoubleClick function inside the component.

One of the main advantages of React is it's virtual tree so we want to avoid modifying the DOM directly like this: `const dblClickBtn = document.getElementById('dblClickBtn');`.